applications/asset_tracker/src/cloud_codec/cloud_codec.h

```diff
@@ -50,6 +50,23 @@ enum cloud_channel {
 	CLOUD_CHANNEL_DEVICE_INFO,
 };
 
+#define CLOUD_CHANNEL_STR_GPS "GPS"
+#define CLOUD_CHANNEL_STR_FLIP "FLIP"
+#define CLOUD_CHANNEL_STR_IMPACT 0
```

Comment:
Why are these strings unspecified? Could we use empty string instead (`""`)? The name indicates to expect a string in this define.

Reply:
i just made them null since they are currently null in the lookup array and not used.
what is your preference?  
setting them to empty or just removing them?

Reply:
If your logic relies on being them NULL, we can stick to it - just please use explicit `NULL` symbol instead of `0`.

Reply:
i'll just remove them since they are not used.